scripts/transitway_engine/parse_config.exs

```diff
@@ -0,0 +1,90 @@
+# This is a temporary script to help port Transitway Engine functionality. It parses the
+# code/config and transforms the data into a format that works for realtime_signs.
+
+# This script currently expects to be run from the root directory of realtime_signs,
+# with a copy of transitway_engine sitting in a sibling directory.
+
+# NOTE: This script will modify/overwrite files, so only run it with a clean working directory.
+
+Mix.install([{:jason, "~> 1.4.0"}])
```

Comment:
Ah, thanks for the tip. I agree in general that we should keep dependencies upgraded, but let's do that separately since this script is temporary anyway.

Reply:
True, we won't be reusing this ever once TWE is fully ported so need to sweat it